### ISSUE TITLE
Consider linking the original project

### ISSUE THREAD

**Issue by bsideup** [herman-rogers/stream-components#4]
Hi! If you hard-fork my [original project](https://github.com/bsideup/rx-connect) please consider at least adding a link to it and mentioning the reasons why you did so. Thanks.

For the future reference - even the README.md has [my CI badges and stuff](https://raw.githubusercontent.com/herman-rogers/stream-components/dde209499cd9c0c36336b1b0b1e35901ca535c00/README.md).

**Comment by herman-rogers** [comment]
Yup - not trying to steal everyone will get credit for their work.
